### ISSUE TITLE
[Settings> Add new profile]: Confirmation message is not announced by Screen Reader when user create a new profile and duplicate the profile.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12038]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win + Ctrl + Enter' keys.
4. Activate 'Add a new profile' tab, then activate 'New Empty profile' and 'Duplicate' button.
5. Observe the issue.

**User Experience:**
Screen Reader users UX will not be good if there will not be able to know whether the action is performed successfully or not.

**Guideline Reference:**
EN 301 549 V3.2.1: 11.5.2.15 Change notification

**Attachments:**
[Confirmation message is not announced by Screen Reader when user create a new profile and Duplicate the profile..zip](https://github.com/microsoft/terminal/files/7778772/Confirmation.message.is.not.announced.by.Screen.Reader.when.user.create.a.new.profile.and.Duplicate.the.profile.zip)

### Expected Behavior

Confirmation message should be announced by Screen Reader when user create a new profile and duplicate the profile.
i.e. When New profile created successfully, Screen Reader should '<profile name> created successfully' and When user duplicated a profile Screen Reader should announce '<profile name> created successfully'.

### Actual Behavior

Confirmation message is not announced by Screen Reader when user create a new profile and duplicate the profile.

**Observation:** Issue repro with NVDA and JAWS Screen Reader.

**Comment by carlos-zamora** [comment]
Since we can't fail to create a new profile, this should be pretty easy. Just after invoking the button, send a notification event saying "profile created successfully".
